server/src/controllers/routineController.ts

```diff
@@ -0,0 +1,94 @@
+import { 
+  RoutineApiResponse,
+  RoutinesApiResponse,
+  RoutineDataResponse
+} from "../models/io/routines";
+
+import { 
+  getRoutineById,
+  getAllRoutines,
+  assignRoutineToMember,
+  unassignRoutineFromMember,
+  getRoutinesByMemberId
+} from "../data/routineData";
+
+export async function generateRoutineByIdGetResponse(
+  id: number
+): Promise<RoutineApiResponse> {
+  try {
+    const routine = await getRoutineById(id);
+    let res: RoutineApiResponse = {
+      message: `success`,
+      status: 200,
+      data: routine.data,
+    }
+    return res;
+  } catch (e) {
+    return { message: "Could not find routine", status: 404, data: null };
+  }
+}
+
+export async function generateAllRoutinesGetResponse(): Promise<RoutinesApiResponse> {
+  try {
+    const routines = await getAllRoutines();
+    let res: RoutinesApiResponse = {
```

Comment:
From comment in `routineData.ts`

`routines` would return what you have in `routines.data`

So you can change it to:

```js
{
    message: `success`,
    status: 200,
    data: routines
}
```